executor/builder.go

```diff
@@ -2415,6 +2415,11 @@ func buildTableReq(b *executorBuilder, schemaLen int, plans []plannercore.Physic
 	}
 	ts := plans[0].(*plannercore.PhysicalTableScan)
 	tbl, _ := b.is.TableByID(ts.Table.ID)
+	isPartition, physicalTableID := ts.IsPartition()
```

Comment:
It seems that the same operation will be done in builder:2446. Will it raise some problem?

Reply:
It seems the function `buildNoRangeIndexLookUpReader` will not affect the function `buildTableReq`.

Reply:
Good Catch! I will fix it.